Assets/My Assets/Scripts/InteractionBasedTeleporter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractionBasedTeleporter : MonoBehaviour
+{
+    private Transform currentPlayerPosition;
+    private Transform teleportationTarget;
+    //private AudioClip activationSFX; <-- For later use
+
+    protected virtual void SetTeleportationTarget(Transform target)
+    {
+        //Use the script that will be attached to the teleporter objects to set this on Start or Awake
+        teleportationTarget = target;
+    }
+
+    //public virtual void TeleportPlayer()
```

Comment:
Not an issue, but if you do /* at the top, and */ at the bottom of the area you want to be commented out it saves you a lot of trouble.